core/src/test/kotlin/ch/ergon/dope/QueryBuilderTest.kt

```diff
@@ -685,7 +685,7 @@ class QueryBuilderTest {
 
     @Test
     fun `should Support Not 3`() {
-        val expected = "SELECT NOT(TRUE AND FALSE AND TRUE)"
+        val expected = "SELECT NOT((TRUE AND (FALSE AND TRUE)))"
```

Comment:
As already discussed: remove the brackets from NOT

Reply:
The redundant brackets are removed from Prefix expressions; the space persists with NOT expressions but is removed from Negation expressions.